sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -342,6 +344,7 @@ SUBROUTINE CALRH_PW(RHPW)
         k=lm-l+1
        DO J=JSTA,JEND
         DO I=1,IM
+          if(q(i,j,k)<spval) then
```

Comment:
I suggest add checks of PMID, PW and T in line 347.

Reply:
will do